dashboard-ui/src/components/App/index.jsx

```diff
@@ -149,9 +150,14 @@ export class App extends React.Component {
                                 <li className="nav-item">
                                     <Link className="nav-link" to="/">Home</Link>
                                 </li>
-                                <li className="nav-item">
-                                    <Link className="nav-link" to="/results">Results</Link>
-                                </li>
+                                <NavDropdown title="Results">
+                                    <NavDropdown.Item as={Link} lassName="dropdown-item" to="/results">
+                                        Alignment Results
+                                    </NavDropdown.Item>
+                                    <NavDropdown.Item as={Link} lassName="dropdown-item" to="/dataset">
+                                        Aggregate Results
```

Comment:
Can you change results back to how it was and add something on the end of the nav that says "Data Analysis" or something like that.  I want to keep your aggregate results separated because it will include all the other drop down results in yours.  Delegation/ADM/Text based

Reply:
Should be fixed now